src/rfac/rfr1.c

```
@@ -0,0 +1,82 @@
+ * \param[in] eng pointer to list of energy values.
+ * \param[in] e_int pointer to list of experimental intensity values.
+ * \param[in] t_int pointer to list of theoretical intensities.
+ *
+ * \note \p eng must be terminated by #F_END_OF_LIST.
+ *
+ * \retval R1-factor, if successful.
+ * \retval F_FAIL, if failed.
+ */
+real rfac_r1(const real *eng, const real *e_int, const real *t_int)
+{
+  int i_eng, n_eng;
```

Clang (14.0.6)

[warning] multiple declarations in a single statement reduces readability

(readability-isolate-declaration)